src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -124,7 +124,6 @@ public sealed partial class BestFitMappingAttribute : System.Attribute
         public bool BestFitMapping { get { throw null; } }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-    [System.ObsoleteAttribute("BStrWrapper and support for marshalling to the VARIANT type may be unavailable in future releases.")]
```

Comment:
Is this related to any known issue - mention it in the PR?

Reply:
Ok, I guess this commit just accidentally got into this one from the other PR.

Reply:
Removed offending commit.